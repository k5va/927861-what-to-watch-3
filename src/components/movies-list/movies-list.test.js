import MoviesList from "./movies-list";

const movies = [
  {
    title: `Movie 1`,
    src: `pic 1`
  },
  {
    title: `Movie 2`,
    src: `pic 2`
  },
  {
    title: `Movie 3`,
    src: `pic 3`
  }
];
const handleMovieTitleClick = () => { };

it(`MoviesList should render correctly`, () => {
  const renderedTree = renderer
    .create(
        <MoviesList movies={movies} onMovieTitleClick={handleMovieTitleClick} />)
    .toJSON();
  expect(renderedTree).toMatchSnapshot();
});
