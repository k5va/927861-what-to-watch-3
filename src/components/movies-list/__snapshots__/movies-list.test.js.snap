// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesList should render correctly 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onHover={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Movie 1"
        height="175"
        src="pic 1"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
        onClick={[Function]}
      >
        Movie 1
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onHover={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Movie 2"
        height="175"
        src="pic 2"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
        onClick={[Function]}
      >
        Movie 2
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onHover={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Movie 3"
        height="175"
        src="pic 3"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
        onClick={[Function]}
      >
        Movie 3
      </a>
    </h3>
  </article>
</div>
`;
